test/js/css-image-value.js

```diff
@@ -0,0 +1,34 @@
+suite('CSSImageValue', function() {
+  test('CSSImageValue object is a CSSImageValue, CSSResourceValue, and CSSStyleValue', function() {
+    assert.instanceOf(new typedOM.internal.CSSImageValue(new Image()), typedOM.internal.CSSImageValue);
```

Comment:
No, it's that this test should be

```javascript
assert.instanceOf(new typedOM.internal.CSSImageValue(new Image()), CSSImageValue);
```

so that it's more consistent with what a developer would see when they do an instanceof check.